distribution/lib/Standard/Base/0.0.0-dev/src/Data/Regression.enso

```diff
@@ -103,7 +103,8 @@ type Fitted_Model
         Fitted_Model.Logarithmic a b _ -> a.to_text + " * X.ln + " + b.to_text
         Fitted_Model.Power a b _ -> a.to_text + " * X ^ " + b.to_text
 
-    ## Use the model to predict a value.
+    ## GROUP Math
```

Comment:
Given it's more complicated, my intuition leans that maybe this should be in `Calculations`?